test_fiscalyear.py

```diff
@@ -395,6 +403,23 @@ def test_q3(self, a):
     def test_q4(self, a):
         assert a.q4 == fiscalyear.FiscalQuarter(2016, 4)
 
+    def test_is_leap(self, a, b, g):
+        # default US start_year='previous', start_month=10
+        assert a.isleap is True
+        assert g.isleap is False
+
+        with fiscalyear.fiscal_calendar(start_year='previous', start_month=1):
+            assert a.isleap is False
+            assert b.isleap is True
+
+        with fiscalyear.fiscal_calendar(start_year='same', start_month=3):
+            assert a.isleap is False
+            assert g.isleap is True
+
+        with fiscalyear.fiscal_calendar(start_year='same', start_month=1):
+            assert a.isleap is True
+            assert g.isleap is False
```

Comment:
I'm always hesitant when I see `is` used in Python. Personally, I only use `is` to compare against `None`. I would instead do:
```suggestion
    def test_is_leap(self, a, b, g):
        # default US start_year='previous', start_month=10
        assert isinstance(a.isleap, bool)
        assert isinstance(g.isleap, bool)

        with fiscalyear.fiscal_calendar(start_year='previous', start_month=1):
            assert not a.isleap
            assert b.isleap

        with fiscalyear.fiscal_calendar(start_year='same', start_month=3):
            assert not a.isleap
            assert g.isleap

        with fiscalyear.fiscal_calendar(start_year='same', start_month=1):
            assert a.isleap
            assert not g.isleap
```